### PR TITLE
Improve copy

### PR DESCRIPTION
**Pull request by deizel**
Noticed a few typos so did some proofreading, hope you don't mind. :wink:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -201,7 +195,7 @@ <h4 class="list-group-item-heading"><code>your-repo.git</code>/<code>branch.requ
 					</a>
 				</p>
 				<p class="text-right">
-					<copy>© <a href="//noelboss.com">Noël Bossart</a>. Made in Switzerland.</copy>
+					<small>© <a href="//noelboss.com">Noël Bossart</a>. Made in Switzerland.</small>
```
Comment (deizel) [inline index.html:198]: Oops, I forgot to add `class="copy"` here. :disappointed: (vim complained that `<copy>` wasn't a tag)

**Comment by noelboss** [comment]
Hey Chris, thanks for the work you've done!

File: index.html (hunk 1 above)
Comment (noelboss) [inline index.html:198]: I've added it do develop

**Comment by deizel** [comment]
No problem :+1:

File: index.html (hunk 1 above)
Comment (deizel) [inline index.html:198]: Great!

**Pull request merged**